### PR TITLE
EDITORIAL CLOSING: XMPP Newsletter July 2022

### PR DESCRIPTION
**Pull request by Echolon**
## This is the draft for the next release covering July 2022.

Final release is planned for 04 of August 2022.

You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: [commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)

[You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links pray

Just paste your prepared text in this thread below or in our XMPP channel (xmpp:[commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)?join)

Please do not just drop bare links - rather try to make a short and decent formulated description (<5 lines) what is the main outcome of your source. You do not need to write in a hard, formal tone - be passionate why this motivates you or what you work on!

Status:

* [x] Collection of news
* [x] Review
* [ ] Blog post + Tiny Newsletter
* [ ] Social media
* [ ] Translations
* [ ] Archived ([Use pandoc!](https://pandoc.org/))

### CODE REVIEW

File: content/blog/2022-08-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+The XEP development process starts by writing up an idea and submitting it to the XMPP Editor. Within two weeks, the Council decides whether to accept this proposal as an Experimental XEP.
+
+-   [WebSocket S2S](https://xmpp.org/extensions/inbox/websocket-s2s.html)
+    -   This specification defines a procedure to make s2s XMPP connections over WebSocket.
+-   [XMPP over QUIC](https://xmpp.org/extensions/inbox/xmpp-over-quic.html)
+    -   This specification defines a procedure to make both c2s and s2s XMPP connections over the QUIC protocol instead of TCP+TLS.   
+
+### New
+
```
Comment (Echolon) [inline content/blog/2022-08-05-newsletter.md:122]: Add RFC 9266

**Comment by SamWhited** [comment]
Could we add something along the lines of:

> The Mellium Dev Communiqué for July 2022 has been published! Important updates this month include an implementation of the new RFC 9266 and a new tool for finding documentation related issues in Go projects:
>
> https://opencollective.com/mellium/updates/dev-communique-for-july-2022

File: content/blog/2022-08-05-newsletter.md (hunk 1 above)
Comment (Echolon) [reply content/blog/2022-08-05-newsletter.md:122]: Mistaken info

File: content/blog/2022-08-05-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+## Events
+
+- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
+- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month
+
+## Articles
+
+In general, articles and thoughts on the European Digital Markets Act:
+- https://ec.europa.eu/info/strategy/priorities-2019-2024/europe-fit-digital-age/digital-markets-act-ensuring-fair-and-open-digital-markets_en
```
Comment (SamWhited) [inline content/blog/2022-08-05-newsletter.md:52]: Maybe replace these with the titles/authors and link them? Having bare URLs in the newsletter feels weird.

File: content/blog/2022-08-05-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+- https://www.zdf.de/nachrichten/politik/whatsapp-threema-interoperabilitaet-100.html
+- https://www.freie-messenger.de/en/begriffe/interoperabilitaet/dma
+- https://www.freie-messenger.de/en/begriffe/interoperabilitaet/gedanken
+
+The site "Chat standard (XMPP)" with lots of information is avilable in Englisch now at [freie-messenger.de](https://www.freie-messenger.de/en/sys_xmpp)
+
+Axel from eversten.net published an [XMPP Push Notification Test for iOS](https://eversten.net/en/blog/notification/) covering latest Monal, Siskin IM and Snikket versions on different
```
Comment (SamWhited) [inline content/blog/2022-08-05-newsletter.md:59]: Linkify `eversten.net`?

File: content/blog/2022-08-05-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+- https://www.freie-messenger.de/en/begriffe/interoperabilitaet/gedanken
+
+The site "Chat standard (XMPP)" with lots of information is avilable in Englisch now at [freie-messenger.de](https://www.freie-messenger.de/en/sys_xmpp)
+
+Axel from eversten.net published an [XMPP Push Notification Test for iOS](https://eversten.net/en/blog/notification/) covering latest Monal, Siskin IM and Snikket versions on different 
+server implementations. A good 15 mins read for both iOS users and their multi-platforms contacts alike.
+
+JMP now delivers transcribed voicemail in many common languages, not just English. The [voicemail system has also been updated](https://blog.jmp.chat/b/july-newsletter-2022) to source names from more modern XMPP profiles including vCard4 and PEP Nickname.
+
+MangooseIM 5.1 was released last month, as announced in the [June 2022 Newsletter](https://xmpp.org/2022/07/the-xmpp-newsletter-june-2022/), and that brought improvements to the Inbox features, too. You can now read more about the new Inbox settings and scalability improvements in this [article](https://www.erlang-solutions.com/blog/updates-to-the-mim-inbox-in-version-5-1/).
```
Comment (SamWhited) [inline content/blog/2022-08-05-newsletter.md:64]: I'd split this into two sentences, it's a bit of a runon. "MoongoseIM was released as announced. In addition to the previously mentioned features, this brought improvements to the Inbox feature. …"

File: content/blog/2022-08-05-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+## Software news
+
+### Clients and applications
+
+The Mellium Dev Communiqué for July 2022 has been published! Important updates this month include an [implementation of the new RFC 9266 and a new tool for finding documentation related issues in Go projects](https://opencollective.com/mellium/updates/dev-communique-for-july-2022). Besides, Mellium has moved to Codeberg! The repositories can now be found at [codeberg.org/mellium](https://codeberg.org/mellium/) and the announcement post can be found on [Lemmy](https://community.xmpp.net/post/34677) or [Open Collective](https://opencollective.com/mellium/updates/mellium-development-has-moved-to-codeberg).
```
Comment (SamWhited) [inline content/blog/2022-08-05-newsletter.md:70]: The links appear to have gotten changed and don't make sense anymore, the checkdoc tool is here: https://mellium.im/checkdoc/

s/Besides,/In addition,/

**Review by SamWhited — CHANGES_REQUESTED** [review]

File: content/blog/2022-08-05-newsletter.md (hunk 3 above)
Comment (wurstsalat3000) [inline content/blog/2022-08-05-newsletter.md:59]:
```suggestion
Axel from [eversten.net](https://eversten.net) published an [XMPP Push Notification Test for iOS](https://eversten.net/en/blog/notification/) covering latest Monal, Siskin IM and Snikket versions on different 
```

File: content/blog/2022-08-05-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+- [Mellium Co-op](https://opencollective.com/mellium)
+
+## Events
+
+- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
+- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month
+
+## Articles
+
+In general, articles and thoughts on the European Digital Markets Act:
```
Comment (wurstsalat3000) [inline content/blog/2022-08-05-newsletter.md:51]:
```suggestion
There are some articles and thoughts on the European Digital Markets Act: 

- [ec.europa.eu](https://ec.europa.eu/info/strategy/priorities-2019-2024/europe-fit-digital-age/digital-markets-act-ensuring-fair-and-open-digital-markets_en)
- [ZDF about interoperatibility](https://www.zdf.de/nachrichten/politik/whatsapp-threema-interoperabilitaet-100.html) [DE]
- German [freie-messenger.de about digital markets act](https://www.freie-messenger.de/en/begriffe/interoperabilitaet/dma)
- German [freie-messenger.de about interoperatibility](https://www.freie-messenger.de/en/begriffe/interoperabilitaet/gedanken)
```

File: content/blog/2022-08-05-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+
+[Conversations 2.10.8](https://conversations.im/), a Jabber/XMPP client for Android, has been released.
+
+[Gajim 1.4.6](https://gajim.org/post/2022-07-07-gajim-1.4.6-released/) and [1.4.7](https://gajim.org/post/2022-07-24-gajim-1.4.7-released/) have been released. These releases bring performance improvements, better file previews, and fix some bugs with the status icon and notifications. Emoji short code detection has been improved as well.
+
+Monal [version 5.2.0](https://github.com/monal-im/Monal/releases/tag/Build_iOS_796) has been release by the Monal development team! This release highlights are the new (private) push servers in Europe and US, display of transferred files in app's documents directory and several fixes on its encryption (OMEMO), QR code scanning and onboarding. Besides, Monal has also a [new webpage](https://www.monal-im.org/).
```
Comment (wurstsalat3000) [inline content/blog/2022-08-05-newsletter.md:76]:
```suggestion
Monal [version 5.2.0](https://github.com/monal-im/Monal/releases/tag/Build_iOS_796) has been released by the Monal development team! This release’s highlights are the new (private) push servers in Europe and the US, display of transferred files in app's documents directory, and several fixes regarding encryption (OMEMO), QR code scanning, and onboarding. Besides, Monal also features a [new webpage](https://www.monal-im.org/).
```

File: content/blog/2022-08-05-newsletter.md (hunk 2 above)
Comment (Echolon) [reply content/blog/2022-08-05-newsletter.md]: 3ac5bc9d17b43d559f3c0e8600f35f26ae81b487

File: content/blog/2022-08-05-newsletter.md (hunk 3 above)
Comment (Echolon) [reply content/blog/2022-08-05-newsletter.md]: 3ac5bc9d17b43d559f3c0e8600f35f26ae81b487

File: content/blog/2022-08-05-newsletter.md (hunk 4 above)
Comment (Echolon) [reply content/blog/2022-08-05-newsletter.md]: 3ac5bc9d17b43d559f3c0e8600f35f26ae81b487

File: content/blog/2022-08-05-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+
+MoongoseIM 5.1 was released as announced, as announced in the [June 2022 Newsletter](https://xmpp.org/2022/07/the-xmpp-newsletter-june-2022/). In addition to the previously mentioned features, this brought improvements to the Inbox feature. You can now read more about the new Inbox settings and scalability improvements in this [article](https://www.erlang-solutions.com/blog/updates-to-the-mim-inbox-in-version-5-1/).
+
+## Software news
+
+### Clients and applications
+
+The Mellium Dev Communiqué for July 2022 has been published! Important updates this month include an [implementation of the new RFC 9266 and a new tool for finding documentation related issues in Go projects](https://opencollective.com/mellium/updates/dev-communique-for-july-2022). In addition, Mellium has moved to Codeberg! The repositories can now be found at [codeberg.org/mellium](https://codeberg.org/mellium/) and the announcement post can be found on [Lemmy](https://community.xmpp.net/post/34677) or [Open Collective](https://opencollective.com/mellium/updates/mellium-development-has-moved-to-codeberg).
```
Comment (SamWhited) [inline content/blog/2022-08-05-newsletter.md:75]: Ah, my mistake, when I suggested the text we were talking about earlier I didn't put the links in. This long link looks really confusing to me though. Maybe something like this, instead:

> [The Mellium Dev Communiqué for July 2022](https://opencollective.com/mellium/updates/dev-communique-for-july-2022) has been published! Important updates this month include an implementation of the new [RFC 9266](https://www.rfc-editor.org/info/rfc9266) and a [new tool](https://mellium.im/checkdoc/) for finding documentation related issues in Go projects

Thanks, and sorry for the confusion!

**Comment by Echolon** [comment]
## Editorial closing of the XMPP Newsletter July 2022 release

- Many thanks to everyone!!! This release to **Adrien Bourmault (neox), anubis, Anoxinon e.V., Benoît Sibaud, cpm, daimonduff, emus, Holger, *IM*, Ludovic Bocquet, martin, MattJ, MSavoritias (fae,ve), nicfab, Pierre Jarillon, Sam Whited, TheCoffeMaker, wh0nix, vanitasvitae, wurstsalat, Zash** for their help in creation, review and translation!

- :heart: :fireworks:  Next release is next month on **5th of September 2022**!

Have a good time until then and see you soon!

**Pull request merged**